test.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
-var assert = require('assert');
-var arrayDifference = require('./');
+import test from 'ava';
+import arrayDifference from './';
 
-it('should filter out the difference', function () {
-	assert.deepEqual(arrayDifference([2, 3, 4], [3, 50, 60]), [2, 4]);
+test('should filter out the difference', t => {
```

Comment:
Yes I know, we let @sindresorhus decide on this :). But the thing is that the tests where written in `mocha` and the convention in `mocha` is that you write `it should filter out the difference`. In `AVA`, this would be `test('filter out the difference'`. And my opinion is that if you only have one test, you can just drop the message.